dashboard/app/models/script_level.rb

```diff
@@ -136,12 +136,8 @@ def end_of_script?
   end
 
   def long_assessment?
-    if assessment
-      if level.properties["pages"] && level.properties["pages"].length > 1
-        return true
-      end
-    end
-    false
+    return false unless assessment
```

Comment:
At this point I find the original a bit easier to read, but i'm biased so maybe a straw poll needed.